model-providers/hugging-face/runtime/src/main/java/io/quarkiverse/langchain4j/huggingface/runtime/config/ChatModelConfig.java

```diff
@@ -44,6 +44,7 @@ public interface ChatModelConfig {
     /**
      * If set to {@code false}, the return results will not contain the original query making it easier for prompting
      */
+    @WithDefault("false")
     Optional<Boolean> returnFullText();
```

Comment:
If a default value is specified, it's a good idea to remove the `Optional`

```suggestion
    boolean returnFullText();
```

Reply:
Also, a better PR title would make it look great in the release notes 😉